src/displayapp/screens/WatchFaceBinary.cpp

```diff
@@ -0,0 +1,693 @@
+	lv_obj_align(dayLED1, dayLED2, LV_ALIGN_OUT_RIGHT_MID, LED_SPACE_H2, 0);		
+	
+	dayLED0 = lv_obj_create(lv_scr_act(), nullptr);
+	lv_obj_set_style_local_bg_color(dayLED0, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_DAY_OFF);
+	lv_obj_set_style_local_radius(dayLED0, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_RADIUS_CIRCLE);
+	lv_obj_set_style_local_border_width(dayLED0, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_RING2);
+	lv_obj_set_style_local_line_color(dayLED0,LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_RING);		
+	lv_obj_set_size(dayLED0, LED_SIZE2, LED_SIZE2);
+	lv_obj_align(dayLED0, dayLED1, LV_ALIGN_OUT_RIGHT_MID, LED_SPACE_H2, 0);		
+	
+	// for months //
+	///////////////
+	
+
+
+	monLED3 = lv_obj_create(lv_scr_act(), nullptr);
+	lv_obj_set_style_local_bg_color(monLED3, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_MON_OFF);
+	lv_obj_set_style_local_radius(monLED3, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_RADIUS_CIRCLE);
+	lv_obj_set_style_local_border_width(monLED3, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_RING2);
+	lv_obj_set_style_local_line_color(monLED3,LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_RING);			
+	lv_obj_set_size(monLED3, LED_SIZE2, LED_SIZE2);
+	lv_obj_align(monLED3, minLED3, LV_ALIGN_OUT_BOTTOM_MID, 0, (LED_SPACE_V2));	// 1/4 Kreis unter dayLED4	
+	
+	monLED2 = lv_obj_create(lv_scr_act(), nullptr);
+	lv_obj_set_style_local_bg_color(monLED2, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_MON_OFF);
+	lv_obj_set_style_local_radius(monLED2, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_RADIUS_CIRCLE);
+	lv_obj_set_style_local_border_width(monLED2, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_RING2);
+	lv_obj_set_style_local_line_color(monLED2,LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_RING);			
+	lv_obj_set_size(monLED2, LED_SIZE2, LED_SIZE2);
+	lv_obj_align(monLED2, monLED3, LV_ALIGN_OUT_RIGHT_MID, LED_SPACE_H2, 0);		
+	
+	monLED1 = lv_obj_create(lv_scr_act(), nullptr);
+	lv_obj_set_style_local_bg_color(monLED1, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_MON_OFF);
+	lv_obj_set_style_local_radius(monLED1, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_RADIUS_CIRCLE);
+	lv_obj_set_style_local_border_width(monLED1, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_RING2);
+	lv_obj_set_style_local_line_color(monLED1,LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_RING);			
+	lv_obj_set_size(monLED1, LED_SIZE2, LED_SIZE2);
+	lv_obj_align(monLED1, monLED2, LV_ALIGN_OUT_RIGHT_MID, LED_SPACE_H2, 0);		
+	
+	monLED0 = lv_obj_create(lv_scr_act(), nullptr);
+	lv_obj_set_style_local_bg_color(monLED0, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_MON_OFF);
+	lv_obj_set_style_local_radius(monLED0, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_RADIUS_CIRCLE);
+	lv_obj_set_style_local_border_width(monLED0, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_RING2);
+	lv_obj_set_style_local_line_color(monLED0,LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LED_COL_RING);			
+	lv_obj_set_size(monLED0, LED_SIZE2, LED_SIZE2);
+	lv_obj_align(monLED0, monLED1, LV_ALIGN_OUT_RIGHT_MID, LED_SPACE_H2, 0);	
```

Comment:
You can probably use lv_style_t and/or loops to minimize this code.

Reply:
im just a hobby-coder, i can try it, but it will take some time, because i don't know how to do this.
Loop in general are no problem, but how could i address an object by a count variable?

Reply:
You could have an array for all similar objects. You can reference List.cpp and SettingWakeUp.cpp if you'd like to try. SettingWakeUp isn't looped, but it uses an array. List is looped.

https://github.com/JF002/InfiniTime/blob/develop/src/displayapp/screens/List.cpp
https://github.com/JF002/InfiniTime/blob/develop/src/displayapp/screens/settings/SettingWakeUp.cpp